docs/modules/ROOT/pages/references/architecture/control-api-design.adoc

```diff
@@ -0,0 +1,181 @@
+
+API object field: Status::
+The `status` field (also called https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#status-subresource["Status subresource"]) represents the current status and external information.
+It is read-only to the end-user.
+
+API object field: Spec::
+The `spec` field contains the desired state for reconciliation.
+
+=== Virtual vs. CRD based resources
+
+Some resources not persisted to etcd and are only available virtually (API extension), others are persisted to etcd and are defined and represented via `CustomResourceDefinition`s (CRDs).
+
+Virtual resources are accessible via an https://kubernetes.io/docs/tasks/extend-kubernetes/setup-extension-api-server/[API Server Extension].
+These resources are similar to views in a relational database.
+The benefit of providing these resources instead of only using CRDs is that we can calculate access permissions dynamically for every request.
+The same concept is also used by OpenShift with its `Project` resource which represents RBAC filter `Namespaces` (see https://github.com/openshift/kube-projects[kube-projects]).
+
+=== Authentication and Authorization
+
+Authentication against the API server is done by the {idp} by leveraging OIDC.
+This allows to transport authorized information about the user in a JWT token.
+
+For authorization, standard https://kubernetes.io/docs/reference/access-authn-authz/rbac/[Kubernetes RBAC] is being used.
+Kyverno policies can be used to implement enhanced policies, for example the number of resources of a specified kind a user is allowed to create.
+
+== Main Use-Cases
+
+All use-cases are from the view of an API user.
+
+Authentication and Authorization::
+* Authenticate against the API, so that the API knows who I am
+* Authorization to work with resources granted to me
+
+Organizations::
+* List organizations I have access to
+* Create a new organization if I'm allowed to
+* Update an existing organization if I'm allowed to
+* Delete an existing organization if I'm allowed to
+* Invite a user to an organization if I'm allowed to send invitations
+* Join an existing organization If I'm invited
+
+Zones::
+* List of zones I have access to
+* See details of zone I have access to
+
+== Global Resources
+
+These are the resources which are global to the {product} instance.
+They are available in the global scope of the Kubernetes API server.
+
+=== `Organization` (Virtual)
+
+Organizations are the container for all resources which belong to an organization.
+It is represented by a Kubernetes Namespace with specific labels and annotations.
+This allows to use standard Kubernetes RBAC rules for organization-specific resources.
+
+To circumvent the current limitation of Kubernetes to show filtered lists of cluster-scoped resources based on access rights, the virtual object `Organization` represents an organization.
```

Comment:
Can you rephrase that? I don't understand how `the virtual object Organization represents an organization.` circumvents RBAC limitations?